RHEL/7/input/oval/oval_5.11/service_rpcgssd_disabled.xml

```diff
@@ -0,0 +1,42 @@
+<def-group>
+
```

Comment:
Based on:
&nbsp; &nbsp; [1] https://bugzilla.redhat.com/show_bug.cgi?id=1173564#c1

the ```rpc.gssd.service``` in RHEL-7 is part of ```nfs-secure.service```, which is part of ```nfs-client.target```. So instead of checking ```service_rpcgssd_disabled / status``` (for which systemd will return value of ```static```), we should probably check status of ```nfs-client.target``` (if it's enabled / disabled). This can be implemented in future PRs, for now I suggest just to remove the rpcgssd entry from this PR.